alert_source.go

```diff
@@ -0,0 +1,57 @@
+package opslevel
+
+import (
+	"github.com/shurcooL/graphql"
+)
+
+type AlertSourceExternalIdentifier struct {
+	Type 			 AlertSourceTypeEnum	`graphql:"type" json:"type"`
+	ExternalId string 							`graphql:"externalId" json:"externalId"`
+}
+
+type AlertSource struct {
+	Name           string
+	Description    string
+	Id             graphql.ID
+	Type           AlertSourceTypeEnum
+	ExternalId     string
+	Integration    Integration
+	Url            string
+	Metadata       string
```

Comment:
Is there a size limit on metadata?  Looking at the metadata returned in our circular account it seems like this is probably not going to be generally useful to the CLI and Terraform.  So unless there is a good reason i'd recommend removing this field as it could potentially be alot of data to pull back from the API - especially once we implement the `List` function.